app/modules/main/directives/header/mdtColumnDirective.js

```diff
@@ -66,7 +68,8 @@
                 columnDefinition: '@',
                 columnSort: '=?',
                 columnFilter: '=?',
-                excludeFromColumnSelector: '=?'
+                excludeFromColumnSelector: '=?',
```

Comment:
This is used to hide the column from the column selector. I will rename the new  property to make it more clear.